wallet/src/de/schildbach/wallet/ui/OnboardingActivity.kt

```diff
@@ -252,8 +251,4 @@ class OnboardingActivity : RestoreFromFileActivity() {
             finish()
         }
     }
-
-    fun getWalletApplication() : WalletApplication {
-        return walletApplication
```

Comment:
https://console.firebase.google.com/u/1/project/dash-wallet-firebase/crashlytics/app/android:hashengineering.darkcoin.wallet/issues/07e4099651ae275ff7bd8936f94f0fc8?time=last-seven-days&types=crash&sessionEventKey=62B38FF40390000113013292E94DDA46_1690632123262373420

Called from `RestoreWalletDialogFragment` and ends up being not initialized. If we inject it in there instead, this should fix it.